Assignment/Completed/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+		<title>Sam heiligers</title>
+
+		<link rel="icon" href="favicon.ico" type="image/x-icon"/>
+
+		<link href="./style.css" rel="stylesheet"
+	</head>
+	<body>
+		<table style="width:100%" valign="top">
```

Comment:
Avoid using tables unless it's an actual table of data. You should be able to do things with floats now.